staging/src/k8s.io/kubectl/pkg/describe/describe_test.go

```diff
@@ -698,7 +718,7 @@ func TestDescribeService(t *testing.T) {
 				Port:                     port-tcp  8080/TCP
 				TargetPort:               9527/TCP
 				NodePort:                 port-tcp  31111/TCP
-				Endpoints:                <none>
+				Endpoints:                10.244.0.1:9527,10.244.0.2:9527,10.244.0.3:9527
```

Comment:
ah, so before your refactoring commit, it wasn't testing `formatEndpoints` at all. ok.